build.gradle

```diff
@@ -14,7 +14,7 @@ plugins {
   id 'com.github.ben-manes.versions' version '0.39.0'
   id 'org.sonarqube' version '3.3'
   id "info.solidsoft.pitest" version '1.7.0'
-  id 'au.com.dius.pact' version '4.1.7'
+  id 'au.com.dius.pact' version '4.3.9'
```

Comment:
did you check if the serenity report is being generated for this upgraded version?